.github/workflows/ci.yml

```diff
@@ -31,6 +31,7 @@ jobs:
     name: Check
     runs-on: ubuntu-latest
     steps:
+      - run: sudo apt install -y libprotobuf-dev protobuf-compiler
```

Comment:
Do we need to run "build" command to compile proto file?

Reply:
`build.rs` will compile proto file.